File: mifosng-android/src/main/java/com/mifos/services/data/ClientsAttendance.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

package com.mifos.services.data;


import com.google.gson.Gson;

public class ClientsAttendance
{
    public int attendanceType;
    public int clientId;
    @Override
    public String toString()
    {
        return new Gson().toJson(this);
    }
}

File: core/datastore/src/main/java/com/mifos/core/model/ClientsAttendance.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class ClientsAttendance(

    var attendanceType: Int,

    var clientId: Int
) : Parcelable